frontend/src/components/tabs/RuntimeNodeDetailsV2.tsx

```diff
@@ -233,6 +257,42 @@ function getTaskDetailsFields(
   return details;
 }
 
+function getNodeVolumeMounts(
+  layers: string[],
+  templateString?: string,
+  element?: FlowElement<FlowElementDataBase> | null,
+): Array<KeyValue<string>> {
+  if (!templateString || !element) {
+    return [];
+  }
+
+  const taskKey = getTaskKeyFromNodeKey(element.id);
+  const pipelineSpec = convertYamlToV2PipelineSpec(templateString);
+  const componentSpec = getComponentSpec(pipelineSpec, layers, taskKey);
+  const platformSpec = convertYamlToPlatformSpec(templateString);
+
+  if (!platformSpec) {
+    return [];
+  }
+
+  // Currently support kubernetes platform
+  const k8sDeploymentSpec = PlatformDeploymentConfig.fromJSON(
+    platformSpec.platforms['kubernetes'].deploymentSpec,
+  );
+  const matchedExecutorObj = Object.entries(k8sDeploymentSpec.executors).find(
+    ([executorName]) => executorName === componentSpec?.executorLabel,
+  );
+
+  let volumeMounts: Array<KeyValue<string>> = [];
+  if (matchedExecutorObj) {
+    const executor = KubernetesExecutorConfig.fromJSON(matchedExecutorObj[1]);
+    const pvcMounts = Object.values(executor.pvcMount).map(pvcm => PvcMount.fromJSON(pvcm));
+    volumeMounts = pvcMounts.map(pvcm => [pvcm.mountPath, pvcm.taskOutputParameter?.producerTask]);
```

Comment:
What is the value of `pvcm.taskOutputParameter?.producerTask` if `taskOutputParameter` or `producerTask` is empty? Is there any other field that we can use from `pvcm`?

Reply:
1. It should be empty (undefined)
2. Based on current design, there're two additional fields in pvcm: `componentInputParameter` and `constant`